torch/nn/modules/activation.py

```diff
@@ -271,6 +271,29 @@ def forward(self, input):
         return torch.sigmoid(input)
 
 
+class Hardsigmoid(Module):
+    r"""Applies the element-wise function:
+
+    .. math::
+        \text{Hardsigmoid}(x) = \frac{ReLU6(x + 3)}{6}
```

Comment:
This math takes a little longer to understand. Instead, we can write it as:

```
.. math::
    \text{Hardsigmoid}(x) = \begin{cases}
        0 & \text{if~} x \le -3, \\
        1 & \text{if~} x \ge +3, \\
        x / 6 & \text{otherwise}
    \end{cases}
```

would render into

![image](https://user-images.githubusercontent.com/4216323/76648146-5868b180-651b-11ea-9cfc-607a24bbd06e.png)